System.IO.Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -11,12 +11,7 @@ public class DirectoryInfoWrapper : DirectoryInfoBase
 
         public DirectoryInfoWrapper(DirectoryInfo instance)
         {
-            if (instance == null)
-            {
-                throw new ArgumentNullException("instance");
-            }
-
-            this.instance = instance;
+            this.instance = instance ?? throw new ArgumentNullException("instance");
```

Comment:
`nameof(instance)`  - same for other changed files

Reply:
@fgreinacher fixed!